samples/fpgaperf_counter/fpgaperf_counter.h

```diff
@@ -61,6 +70,28 @@ typedef struct {
 	perf_events_type *perf_events;
 } fpga_perf_counter;
 
+/**
+ * Initialize the pthread mutex.
```

Comment:
I think this is an implementation detail that doesn't really summarize what this function does.
IOW, it's too specific .

Reply:
Or maybe this really needs to be an internal function (not an API function) that is called during the initialization of the fpga_perf_counter structure.